IO2P/resourceAdder.cs

```diff
@@ -8,14 +8,18 @@
 using MongoDB.Driver;
 using System.Net;
 using System.IO;
+using System.Net.Sockets;
+using System.Net.Http;
+using System.Web;
 
 namespace IO2P
 {
     class resourceAdder
     {
-        private String strDatabase = "";
-        private String strDBUser = "";
-        private String strDBPass = "";
+        private String host = "146.185.128.224:27017"
+        private String strDatabase = "io2";
+        private String strDBUser = "io2";
+        private String strDBPass = "sfd9879IjgkDslkhgdgl98sdfG9sdUFSD98sdf9wfdgHG78vgsf809few0";
```

Comment:
Radzę trzymać ustawienia lokalnie w ENV'ach np. Jest to również konieczne jak travis będzie sobie odpalał testy (tam będzie lokalna baza danych postawiona przez travisa i ustawienia do niej będą brane z envów. Pisałem dokładnie jak się nazwywają w tickecie). Teraz będę musiał zmienić hasło bo zna je już cały świat :(